compiler/src/dotty/tools/dotc/core/Flags.scala

```diff
@@ -293,7 +293,11 @@ object Flags {
    */
   val (Abstract @ _, _, _) = newFlags(23, "abstract")
 
-  /** Lazy val or method is known or assumed to be stable and realizable */
+  /** Lazy val or method is known or assumed to be stable and realizable.
+   *
+   *  For a trait constructor, this is set if and only if owner.is(NoInits),
```

Comment:
If it's exactly equivalent, why do we need to set it instead of checking owner.is(NoInits) directly?

Reply:
Because that is what Scala 2's Tasty reader will look at. Also, the inliner looks at this to determine whether it is safe to elide.

Reply:
>  Also, the inliner looks at this to determine whether it is safe to elide.

I think that's worth mentioning in the documentation of this flag.

Reply:
Done.